services/hbase/Dockerfile

```diff
@@ -14,10 +14,21 @@
 #
 
 FROM kilda/base-ubuntu
-ADD ansible /ansible
-# tar should be pre-populated with the tarball; see the top level Makefile (build-base:)
-ADD tar /src
-RUN ansible-playbook -s /ansible/deploy.yml
-ADD hbase-conf/hbase-env.sh /opt/hbase/conf/hbase-env.sh
-ADD hbase-conf/hbase-site.xml /opt/hbase/conf/hbase-site.xml
-ADD hbase-conf/start-hbase /opt/hbase/bin/start-hbase
+
+ENV PACKAGE hbase-1.2.4
```

Comment:
If it was possible it could be great to extract 1.2.4 as an independent variable and use it in commands/scripts below. In case of changing the hbase version, we could just change one variable value instead of updating a bunch of commands/scripts.